src/modules/toDo_UI.js

```diff
@@ -0,0 +1,60 @@
+import Remove from './Remove.js';
+
+export default function display() {
+  const todoList = document.querySelector('#todo-list-wrap');
+  const todos = JSON.parse(localStorage.getItem('todos')) || [];
+
+  todoList.innerHTML = '';
+
+  todos.forEach((todo) => {
+    const todoItem = document.createElement('div');
+    todoItem.classList.add('todo-item');
+
+    const label = document.createElement('label');
+    const input = document.createElement('input');
+    const span = document.createElement('span');
+    const content = document.createElement('div');
+    const actions = document.createElement('div');
+    const editBtn = document.createElement('button');
+    const removeBtn = document.createElement('button');
+
+    input.type = 'checkbox';
+    input.checked = todo.complet;
+
+    span.classList.add('bubble');
+    content.classList.add('todo-content');
+    actions.classList.add('actions');
+    editBtn.classList.add('edit');
+    removeBtn.classList.add('remove');
+
+    content.innerHTML = `<input type="text" value="${todo.content}" readonly>`;
+    editBtn.innerHTML = 'Edit';
+    removeBtn.innerHTML = 'Delete';
+
+    todoList.appendChild(todoItem);
+    todoItem.appendChild(label);
```

Comment:
Even though your code is functioning as expected but there are project requirements that you have to follow first:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20adding%20a%20new%20task%20(add%20a%20new%20element%20to%20the%20array).), Please add a separate function to add a new task.
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20deleting%20a%20task%20(remove%20an%20element%20from%20the%20array).), Please add a separate function for deleting a task.
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20editing%20task%20descriptions.), Please add a separate function for editing a task.

Because you implement most of your code inside the `display` function and implemented the remove functionality as an object inside the `Remove.js` module and that's not what the project requirements suggested.

Reply:
I have already nested a function to edit, and remove inside src/modules toDO

1-UI {function to edit [https://github.com/bernabasy/ToDo_list_list_structure/blob/95b07a9bb596cc354cafa2920ebc7c167cf1c648/src/modules/toDo_UI.js#L43](https://mailgun.airtable.com/c/eJw1T8tqBCEQ_JrxOPgaHwcPgRBY2GNyDnarOy5ODOoc8vdxCIGmugqqHxUdU5pRbZXhJLiQuEkkO045Z5xtVEpD-WopMCqNT1pTHVRYJPW5DQ8lrlgPsjulQwomyoQMgrBaCqFsiAwwKARrSXH7GN99ES8Lf5v1yGM_4ZqeAmL78uD7z-Tv9bV-ltzHH_TRThxni5etVJjNbkC1twCbVYhik-iT55bTCKhx5sHEUEkzrb3hxKOGs8Q-2biWf9zW5xTiLgVp7v_2s_aYZrDH4XO5_voFMlFXaQ) }

2-{function to remove I have done a method inside [src/modules/ Remove function](https://mailgun.airtable.com/c/eJw1TstuwyAQ_BpztBaCAR84RIr6AVHuFmuWmAiHCnCl_n1xq0qr2Zl9zZLlSnPQszKCeeuDMIFFK0AILvgEUhoQ4wzIQRoXtAbtlR8kuFiaw0Tjmne22QkQgkNtlPKzDMbgpMPFCCQviaRjyW6tfdbhch3ER49nbNuB53YXSOXt0NXvzh_5lpcUa_uD2sqxtqPQOZYy9nT1fnFvvxTa89dZr2XtuGd_JKqd3X8b46uyYv9Pv3Kl0P9-7i6m0_YHMI1QBA) and o have done a nested function inside toDo_UI with the following link. [https://github.com/bernabasy/ToDo_list_list_structure/blob/95b07a9bb596cc354cafa2920ebc7c167cf1c648/src/modules/toDo_UI.js#L55](https://mailgun.airtable.com/c/eJw1T7tqxDAQ_Bq7NKu3VKgIhMDBlUkdtGvp7EOOgiQX-fvIhMAyOwOzj4meacPAOG35vPo1cZvm3XPgnHGmQEoLfHGADKQNyRgwq14nCWGvPWCOC5Vj3rxxSZBlxBwCWSuFjTomwTAlFYmrOfut9-82iZeJv4167H078ZoeAmP9Chjaz-Dv5bV85r31P2i9ntTPGi9bLjiaUwgmOETlNJFQkkIK3HGISIZGHkqMtLTD2ioNPMp65tgG69fyj9vyHELclZqr_7_9LC2mEexxhD1ff_0C3_BXBg) }

3. if you look at the link below the function to add and remove will be done with the event listener function. and then in its used in toDO_UI.js to display it

 [ index.js](https://mailgun.airtable.com/c/eJw1TkFuxCAMfE04RoYkQA4cVlr1Bb1HOIaEFQkrIFX7-xJVlazxeGSPxxkuFQc1Sy0YGfJCexaMACG44BOMowbRz4AcRm29UqBIUjeCDblajK5f08F2o1FKrtED-Qk1gEbi0-rF4DwNRDOLZq_1Xbrh0YmPVluo-4X3dRvQ5dOiLT-Nf6ZnWmIo9Q9Kzddar-zutZiwtQfRYk9asjvS162XvDYMJ7nv_lVYNv9-r1Scb2G3w4Z4__oF65pMPA) file. please look at things again.

and give me the way how can i change it also.